product_docs/docs/eprs/7/07_common_operations/11_using_ssl_connections.mdx

```diff
@@ -181,13 +181,17 @@ Before you begin, configure the client for SSL with trigger mode.
 
   The default location of these files is `{user.home}/.postgresql(e.g/var/lib/edb/.postgresql/)`. The file location can be overridden using SSL connection parameters or Postgres SSL environmental variables, see [Setting non-default paths using environment variables](#setting-non-default-paths-using-environment-variables) for more information.
 
+  !!!note
+  If you have installed Replication Server using the RPM install package from the EDB repository, the value of the {user.home} property is `/var/lib/edb/` and you need to create the  `.postgresql` directory in this path to have a valid default client SSL directory path (`/var/lib/edb/.postgresql/`).
+  !!!
+  
   Copy and rename the files:
   ```
-  $ cd /var/lib/pgsql/.postgresql/
+  $ cd /var/lib/edb/.postgresql/
 
   $ cp /var/lib/pgsql/14/data/server.crt postgresql.crt
 
-  $ cp /var/lib/pgsql/14/data/root.crt .
+  $ cp /var/lib/edb/14/data/root.crt .
```

Comment:
The cd command  is correct. (cd /var/lib/edb/.postgresql/) 
The 3 cp commands should have folder /var/lib/pgsql/14/data.

Please change it to -  cp /var/lib/pgsql/14/data/root.crt .